canonical/ubuntu.com
# WD-12058 - rebrand /cloud-computing

## Done

Rebrand /cloud/cloud-computing. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14103.demos.haus/cloud/cloud-computing)
- [figma](https://www.figma.com/design/NpI2wzW86XXPWxsSeomYfy/u.c%2Fcloud?node-id=31-1589&t=JDqOlKdc8TNDstzv-1)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12058](https://warthogs.atlassian.net/browse/WD-12058)

[WD-12058]: https://warthogs.atlassian.net/browse/WD-12058?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
